docs/MANUAL.md

```diff
@@ -966,6 +967,53 @@ then add the following `startupOptions` to Kiwi's `static/config.json` file (see
     },
 ```
 
+## gamja
+
+[gamja](https://sr.ht/~emersion/gamja/) is another web-based IRC client. For a demonstration of its features, you can try the gamja client on the official Ergo IRC network: [ergo.chat/gamja/](https://ergo.chat/gamja/).
+
+Note that with Ergo you do not need the external WebIRC gateway.
+
+If you do not already have a WebSocket listener (i.e. after following the Kiwi instructions above) configure one in your ergo config (ircd.yaml):
+
+```yaml
+        "127.0.0.1:8068":
+            websocket: true
+```
+
+Add the following block to your nginx config / your nginx server block, and, if necessary, adjust the values to match your environment:
+
+```
+        location /gamja {
+                root    /srv/www/gamja;
+                index   index.html;
+        }
+
+        location /socket {
+                proxy_pass http://127.0.0.1:8068;
+                proxy_read_timeout 600s;
+                proxy_http_version 1.1;
+                proxy_set_header Upgrade $http_upgrade;
+                proxy_set_header Connection "Upgrade";
+                proxy_set_header X-Forwarded-For $remote_addr;
+                proxy_set_header X-Forwarded-Proto $scheme;
+        }	
+```
```

Comment:
Maybe reduce this to just "see the previous section for instructions on how to configure a websocket listener"?

Reply:
I get it's redundant, however since it requires the static file location too, I figured it's more user friendly to show everything it needs versus referencing away for some parts, in order for the user to configure gamja without leaving the section. What do you think?

Reply:
I don't think it requires the static file location; the example for Kiwi assumes that the reader already has configuration ready to serve static files (indeed it's probably not typical to have a separate `location` block for serving Kiwi or gamja).

Reply:
It's possible that we should just consolidate these into a single section named something like "Websocket clients (Kiwi and gamja)"...

Reply:
I like the idea of a consolidated section.

Reply:
Re: static files: The gamja instructions suggest the user it is the job of the WebIRC gateway to provide static files. As this is not the case with ergo, I figured it's a good idea to clarify how it needs to be done.